lib/tools/adb-commands.js

```diff
@@ -728,4 +728,19 @@ methods.getScreenSize = async function() {
   return null;
 };
 
+methods.setHttpProxy = async function (proxyHost, proxyPort) {
```

Comment:
@mykola-mokhnach I would leave proxyHost since the adb setting is called `global_http_proxy_host`